File: java/src/main/java/com/meteroid/models/CreditType.java
// This file is @generated
package com.meteroid.models;

import com.fasterxml.jackson.annotation.JsonValue;
import com.meteroid.Utils.ToQueryParam;

public enum CreditType implements ToQueryParam {
    CREDIT_TO_BALANCE("CREDIT_TO_BALANCE"),
    REFUND("REFUND"),
    DEBT_CANCELLATION("DEBT_CANCELLATION");
    private final String value;

    CreditType(String value) {
        this.value = value;
    }

    @JsonValue
    public String getValue() {
        return this.value;
    }

    @Override
    public String toQueryParam() {
        return this.value;
    }
}

File: python/meteroid/models/credit_type.py
# this file is @generated
import enum


class CreditType(str, enum.Enum):
    CREDIT_TO_BALANCE = "CREDIT_TO_BALANCE"
    REFUND = "REFUND"
    DEBT_CANCELLATION = "DEBT_CANCELLATION"

    def __str__(self) -> str:
        return str(self.value)
